IBPSA/Fluid/FMI/ExportContainers/Examples/FMUs/HVACZones.mo

```diff
@@ -43,18 +47,18 @@ block HVACZones
     "Nominal air mass flow rate, increased by factor 1.3 to allow for recovery after temperature setback";
   parameter Modelica.SIunits.TemperatureDifference dTFan = 2
     "Estimated temperature raise across fan that needs to be made up by the cooling coil";
-  parameter Modelica.SIunits.HeatFlowRate QCoiC_flow_nominal=4*
-    (QRooC_flow_nominal + mA_flow_nominal*(TASup_nominal-THeaRecLvg-dTFan)*1006)
-    "Cooling load of coil, taking into account economizer, and increased due to latent heat removal";
+  parameter Modelica.SIunits.HeatFlowRate QCoiC_flow_nominal=
+    mA_flow_nominal*(TASup_nominal-THeaRecLvg-dTFan)*1006+mA_flow_nominal*(wASup_nominal-wHeaRecLvg)*2458.3e3
+    "Cooling load of coil, taking into account outside air sensible and latent heat removal";
 
   /////////////////////////////////////////////////////////
   // Water temperatures and mass flow rates
-  parameter Modelica.SIunits.Temperature TWSup_nominal = 273.15+16
+  parameter Modelica.SIunits.Temperature TWSup_nominal = 285.15
     "Water supply temperature";
-  parameter Modelica.SIunits.Temperature TWRet_nominal = 273.15+12
+  parameter Modelica.SIunits.Temperature TWRet_nominal = 289.15
     "Water return temperature";
   parameter Modelica.SIunits.MassFlowRate mW_flow_nominal=
-    QCoiC_flow_nominal/(TWRet_nominal-TWSup_nominal)/4200
+    -QCoiC_flow_nominal/(TWRet_nominal-TWSup_nominal)/4200
```

Comment:
Hey Michael, thanks for making the pull request. I checked out the code changes in the review and it looks alright. Can you please check if the sign change is exactly as intended here? If you confirm this change, we can do the merge into the master.

Reply:
This sign is correct and leads to `mW_flow_nominal` to be positive.